src/Files.App/Actions/Content/Run/RunWithPowershellAction.cs

```diff
@@ -0,0 +1,47 @@
+﻿using CommunityToolkit.Mvvm.ComponentModel;
+using CommunityToolkit.Mvvm.DependencyInjection;
+using Files.App.Commands;
+using Files.App.Contexts;
+using Files.App.Extensions;
+using Files.App.Helpers;
+using Files.App.Shell;
+using Files.Backend.Helpers;
+using System.Threading.Tasks;
+
+namespace Files.App.Actions
+{
+	internal class RunWithPowershellAction : ObservableObject, IAction
+	{
+		public IContentPageContext context = Ioc.Default.GetRequiredService<IContentPageContext>();
```

Comment:
I think both this and `Context_PropertyChanged(params)` should be marked as private